src/esp/assets/ResourceManager.cpp

```diff
@@ -473,8 +473,13 @@ void ResourceManager::addObjectToDrawables(int objTemplateLibID,
 
 PhysicsObjectAttributes& ResourceManager::getPhysicsObjectAttributes(
     const std::string& objectName) {
+  // check existance here?
```

Comment:
Should existence be verified, or are we willing to assume that it will always exist at this point?

Reply:
For now maybe we can assume since we return a reference, but I think better would be to refactor all `PhysicsObjectAttributes` storage and query to smart pointers so we can check and return `nullptr` on failure. Maybe a follow-up PR?